gateway/ln-gateway/src/db.rs

```diff
@@ -45,3 +46,12 @@ impl_db_record!(
     value = LightningGateway,
     db_prefix = DbKeyPrefix::FederationRegistration,
 );
+
+#[derive(Debug, Clone, Eq, PartialEq, Encodable, Decodable)]
+pub struct GatewaydPublicKey;
```

Comment:
`Gateway` or `Gatewayd`?

Reply:
They're essentially the same thing right now, but I'll change it to `Gateway` just to avoid confusion